teuthology/test/test_worker.py

```diff
@@ -73,7 +73,7 @@ def test_run_job_with_watchdog(self, m_tempfile, m_safe_dump, m_environ,
         m_p.returncode = 0
         m_popen.return_value = m_p
         m_t_config.results_server = True
-        worker.run_job(config, "teuth/bin/path")
+        worker.run_job(config, "teuth/bin/path", False)
```

Comment:
If you end up adding a default value for ``verbose`` to ``run_job()``, I'd encourage either dropping these args or naming them, i.e. ``verbose=False``